bin/metrics-docker-stats.rb

```diff
@@ -206,7 +206,7 @@ def container_tags(container)
     @inspect = docker_api(path)
     tag_list = config[:environment_tags].split(',')
     tag_list.each do |value|
-      tags << @inspect['Config']['Env'].select { |tag| tag.to_s.match(/#{value}=/) }.first.gsub(/#{value}=/, '') + '.'
+      tags << @inspect['Config']['Env'].select { |tag| tag.to_s.match(/#{value}=/) }.first.to_s.gsub(/#{value}=/, '') + '.'
```

Comment:
probably better to actually rescue the `nil` as this will insert a bunch of empty strings into the array:
```ruby
$ irb
irb(main):001:0> a = []
=> []
irb(main):002:0> a[0] = ''
=> ""
irb(main):003:0> a
=> [""]
```

Reply:
Having the same amount of fields, even if the content is nil, will keep the consistency for parsing rules, dashboards, etc. Removing the field can cause unexpected issues with other stuff.

Reply:
`option :environment_tags` is more like an output decorator than a filter

Reply:
that sounds fine, just as long as there is a good reason.